### DISCUSSION TITLE
useSharedValue, does it work with out expo how i can learn more about it is there is way we can use it in Class components please help Thanks

### DISCUSSION THREAD

**Discussion by CodeBunny786** [software-mansion/react-native-reanimated#1753] [Q&A]
useSharedValue,  not working without expo

**Comment by jgonet** [comment]
`useSharedValue` works in the expo. [Here's our docs page about Shared Values](https://docs.swmansion.com/react-native-reanimated/docs/shared-values). It isn't possible to use hooks in class components, you have to wrap those inside functional components.
